client/fresheyes-web/src/app/components/modal/SuccessModal.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from "react";
+import Image from "next/image";
+import checkmarkIcon from "../../assets/checkmark.svg";
+import { usePathname, useSearchParams } from "next/navigation";
+
+export const SuccessModal = () => {
+  const pathname = usePathname();
+  const [isOpen, setIsOpen] = React.useState(false);
+  const params = useSearchParams();
+
+  useEffect(() => {
+    const isInstalled = params.get("installation_id") !== null;
+    if (isInstalled) {
+      setIsOpen(true);
+    }
+  }, [pathname]);
+
+  const removeModal = () => {
```

Comment:
```suggestion
  const closeModal = () => {
```
nit: I think closeMoal is more relatable as a variable name